include/spirv-tools/instructions.hpp.inc

```diff
@@ -0,0 +1,5011 @@
+// THIS FILE IS GENERATED
```

Comment:
Can we generate this during the build and not check it into the repo?

Reply:
The build generates it, and I guess it doesn't need to be checked in.  This file does form part of the API, but now that I think about it again, I can't see a reasonable usage where it doesn't get generated before it's needed.